Formula/lib/liblc3.rb

```diff
@@ -0,0 +1,87 @@
+class Liblc3 < Formula
+  desc "Low Complexity Communication Codec library and tools"
+  homepage "https://github.com/google/liblc3"
+  url "https://github.com/google/liblc3/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "958725e277685f9506d30ea341c38a03b245c3b33852cd074da6c8857525e808"
+  license "Apache-2.0"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+
+  uses_from_macos "python"
+
+  patch do
+    url "https://github.com/google/liblc3/commit/81aae5bcd52fa23a68ca7ae19834ea92d83ed43a.patch?full_index=1"
+    sha256 "3faa8430fbed45f9c1c773aadc86b59dad6ed4a785cf975005eaa627f55b67a3"
+  end
+
+  def install
+    args = %w[
+      -Dtools=true
+      -Dpython=true
+    ]
+
+    system "meson", "setup", "build", *args, *std_meson_args
```

Comment:
```suggestion
    ENV.append "LDFLAGS", "-Wl,-rpath,#{rpath}"
    system "meson", "setup", "build", *args, *std_meson_args
```
This should likely allow you to get rid of the patch (which upstream doesn't seem so keen on anyway).

Reply:
that does not quite work, needs to be part of the meson build

```
  Files with missing rpath:
    /usr/local/Cellar/liblc3/1.1.0/bin/dlc3
    /usr/local/Cellar/liblc3/1.1.0/bin/elc3
```

Reply:
Hm, odd. Let me look into it.